src/pages/Counter/container/CountContainer.js

```diff
@@ -0,0 +1,54 @@
+import { Component } from "react";
+
+import CounterView from "../components/CounterView";
+
+class CounterContainder extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true,
+    };
+  }
+  handleDecrement = () => {
+    if (this.state.countValue > 0) {
+      this.setState((state) => ({
+        isEven:
+          state.countValue - 1 === 0 || (state.countValue - 1) % 2 === 0
```

Comment:
decrementedValue % 2 === 0 will be enough. You don't need to handle case with 0 because 0 % 2 === 0 thats true